addons/profile/profile.php

```diff
@@ -282,9 +282,12 @@ public function sub_page_template() {
 	 * Display user questions page.
 	 */
 	public function question_page() {
-		$user_id                        = ap_current_user_id();
+		$user_id = ap_current_user_id();
+		$paged   = ( get_query_var( 'ap_paged' ) ) ? get_query_var( 'ap_paged' ) : 1;
```

Comment:
Hello @qstudio 
This needs to be sanitized using `max( 1, (int) get_query_var( 'ap_paged' ) )` to make sure number is integer and not less than 1. Thanks

Reply:
Sorry about that.